server/graphql/general/src/queries/name_property.rs

```diff
@@ -29,7 +29,7 @@ pub fn configure_name_properties(
     validate_auth(
         ctx,
         &ResourceAccessRequest {
-            resource: Resource::MutateNameProperties,
+            resource: Resource::ConfigureNameProperties,
```

Comment:
Created another permission for configuring name properties since properties have a 'is remote editable' flag, which means they should be able to edit name properties but not configure them even outside of central